tests/providers/airbyte/hooks/test_airbyte.py

```diff
@@ -87,7 +87,7 @@ def test_wait_for_job_error(self, mock_get_job):
             self.hook.wait_for_job(job_id=self.job_id, wait_seconds=0)
 
         calls = [mock.call(job_id=self.job_id), mock.call(job_id=self.job_id)]
-        assert mock_get_job.has_calls(calls)
+        assert mock_get_job.assert_has_calls(calls, any_order=False)
```

Comment:
Shouldn’t this be

```suggestion
        mock_get_job.assert_has_calls(calls, any_order=False)
```

instead? `assert_has_calls` returns None on success, so asserting its return value is entirely backwards.

Reply:
Yes, you are correct. I will fix this :) Thank you for pointing me in the right direction. I am new to this provider's code base, so taking time to understand what's needed.